pkg/run/install/install_fluent_bit.go

```diff
@@ -0,0 +1,247 @@
+package install
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"html/template"
+	"strings"
+	"time"
+
+	helmv2 "github.com/fluxcd/helm-controller/api/v2beta1"
+	sourcev1 "github.com/fluxcd/source-controller/api/v1beta2"
+	"github.com/weaveworks/weave-gitops/pkg/logger"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const configInputs = `
```

Comment:
💡  Instead of inlining these strings we could embed them using https://pkg.go.dev/embed. Makes the code a little more dense.

Reply:
In the case of this PR, putting (3 of ) them in separate files would increase complexity of the codes.